src/App.js

```diff
@@ -1,18 +1,45 @@
-import NavBar from "./components/NavBar";
-import HomePage from "./components/HomePage";
-import SignInPage from "./components/authentication/SignInPage";
 // TO-DO: Imports are commented out until routing is implemented
 // import SignUpPage from "./components/authentication/SignUpPage";
```

Comment:
Now that routing is implemented these should be removed

Reply:
These are removed on my end. Doesn't the red mean a deletion has occurred?

Reply:
Not the red part (the commented part) lines 4-5.

Reply:
Oh those were Tim's comments. I can remove.